source/loader/ur_loader.cpp

```diff
@@ -25,6 +25,8 @@ namespace loader
             {
                 platforms.emplace_back();
                 platforms.rbegin()->handle = handle;
+            } else {
+                return UR_RESULT_ERROR_MODULE_LINK_FAILURE;
```

Comment:
I don't think we want to completely fail init if one of the libraries doesn't load for some reason. Someone might just happen to have a completely different file with the path we expect.

Once we have logging infrastructure, we can print a warning/info about failing to load a library.

Reply:
Done. Removed this failure.